src/ripple/rpc/impl/ShardArchiveHandler.cpp

```diff
@@ -375,13 +375,17 @@ ShardArchiveHandler::next(std::lock_guard<std::mutex> const& l)
     // sleeps.
     auto const& url{archives_.begin()->second};
     auto wrapper = jobCounter_.wrap([this, url, dstDir](Job&) {
+        auto const ssl = url.scheme == "https";
```

Comment:
I sometimes get confused when I see a bunch of `=` in a single command. Could you add parenthesis so that it's clearer what's going on? `auto const ssl = (url.scheme == "https");`. This doesn't affect functionality, just readability.

Reply:
Done